### PR TITLE
Add workflow files

### PR DESCRIPTION
**Pull request by mustafalii**
## Description

This PR adds workflow files for CI/CD for iOS, Android, and Chrome extension.

### CODE REVIEW

File: .github/workflows/cd_android.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  android_build:
+    name: Build and upload android app draft for internal test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 12.x
```
Comment (admiraladmirable) [inline .github/workflows/cd_android.yml:19]: The version of Java used for the CI is java-version: 17.0.x, should these use the same Java version?

File: .github/workflows/cd_chrome.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        with:
+          node-version: "${{ steps.nvm.outputs.NVMRC }}"
+      - name: Install NPM dependencies
+        run: npm install
+      - name: Build with Webpack
+        run: npm run build
+      - name: Setup Flutter
+        uses: subosito/flutter-action@v2
+        with:
+          channel: 'stable'
```
Comment (admiraladmirable) [inline .github/workflows/cd_chrome.yml:29]: Do we target a specific version of flutter? Pinning the version will reduce the chance of the build suddenly breaking:

```
- uses: subosito/flutter-action@v2
  with:
    flutter-version: '2.8.0'
    channel: 'stable'
```

**Comment by admiraladmirable** [comment]
Doesn't have to be done in this review, but some optimizations that can be added are caching the NPM and Flutter dependency install steps.

Examples:
[setup-node](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#caching-packages-data)
[flutter](https://github.com/subosito/flutter-action/issues/16#issuecomment-574311490)

**Review by admiraladmirable — APPROVED** [review]

File: .github/workflows/cd_android.yml (hunk 1 above)
Comment (mustafalii) [reply .github/workflows/cd_android.yml:19]: Will make the change and request review on the other branch!

File: .github/workflows/cd_chrome.yml (hunk 2 above)
Comment (mustafalii) [reply .github/workflows/cd_chrome.yml:29]: Same as previous comment!

**Review by OlliePorter — APPROVED** [review]
LGTM :shipit:

**Review by scasplte2 — APPROVED** [review]

**Pull request merged**